PCP/src/main/java/PCP/packets/PCPGroupUsersList.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * this is a school project under "The Unlicence".
+ */
+package PCP.packets;
+
+import PCP.*;
+import com.google.gson.Gson;
+import java.nio.charset.*;
+import java.util.*;
+
+
+/**
+ *
+ * @author Alessio789
+ * @author gfurri20
+ */
+public class PCPGroupUsersList implements IPCPpacket 
+{
+    private int type;
+    private int listLenght;
+    private ArrayList<String> listOfUsers;
+    private String jsonContent;
```

Comment:
jsoncontent is supposed to be **calculated** with a method, and not be a variable.